### DISCUSSION TITLE
Can't login to Google Workspace

### DISCUSSION THREAD

**Discussion by EricCartman9969** [zitadel/zitadel#7139] [Announcements]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

Self-hosted

### Database

None

### Describe the problem caused by this bug

Can't login to Google Workspace

### To reproduce

I according to this document at https://zitadel.com/docs/guides/integrate/services/google-workspace#sso-profile-for-your-organization

Google Workspace was configured, but the login failed.

It show :This account cannot be accessed because the login credentials could not be verified

**Comment by hifabienne** [comment]
hei @mffap 
As far as I know you have configured google workspace.
Did you encounter this problem?

**Comment by mffap** [comment]
> hei @mffap As far as I know you have configured google workspace. Did you encounter this problem?

Was working when we wrote the guide. Also I could confirm it working around October in a PoC. Not sure where that message stems from without additional information.

**Comment by EricCartman9969** [comment]
I think it's idp.crt but I'm not sure because I've temporarily deactivated zitadel

**Comment by EricCartman9969** [comment]
I tried reinstalling zitadel and reuploading the certificate to Google Workspace, but the error was the same

**Comment by EricCartman9969** [comment]
> > hei @mffap As far as I know you have configured google workspace. Did you encounter this problem?
> 
> Was working when we wrote the guide. Also I could confirm it working around October in a PoC. Not sure where that message stems from without additional information.

Do I need to provide my config profile?

**Comment by mffap** [comment]
> > > hei @mffap As far as I know you have configured google workspace. Did you encounter this problem?
> > 
> > 
> > Was working when we wrote the guide. Also I could confirm it working around October in a PoC. Not sure where that message stems from without additional information.
> 
> Do I need to provide my config profile?

It would for sure help if you could share your configuration in ZITADEL and Google Workspace. Maybe also a video of the flow failing.

> It show :This account cannot be accessed because the login credentials could not be verified

What is "It"? Is that an error you get after the redirect from Google to ZITADEL in ZITADEL? Where is the error shown and on what step?

**Comment by EricCartman9969** [comment]
> > > > hei @mffap As far as I know you have configured google workspace. Did you encounter this problem?
> > > 
> > > 
> > > Was working when we wrote the guide. Also I could confirm it working around October in a PoC. Not sure where that message stems from without additional information.
> > 
> > 
> > Do I need to provide my config profile?
> 
> It would for sure help if you could share your configuration in ZITADEL and Google Workspace. Maybe also a video of the flow failing.
> 
> > It show :This account cannot be accessed because the login credentials could not be verified
> 
> What is "It"? Is that an error you get after the redirect from Google to ZITADEL in ZITADEL? Where is the error shown and on what step?

Error said after landing zitadel Google workspace, wrong link to https://google.com/a/mydomain/acs

**Comment by EricCartman9969** [comment]
<img width="740" alt="1704190011629" src="https://github.com/zitadel/zitadel/assets/73655702/c1fe169d-e615-456d-9ff2-4cfa54286419">

**Comment by EricCartman9969** [comment]
<?xml version="1.0"?>
<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="google.com/a/<my_domain>">
    <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol urn:oasis:names:tc:SAML:1.1:protocol">
        <md:AssertionConsumerService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location=" https://www.google.com/a/<my_domain>/acs" index="0"/>
    </md:SPSSODescriptor>
</md:EntityDescriptor>

This is my zitadel's SAML config

**Comment by mffap** [comment]
Are you configuring a SSO profile for your organization or Third-party SSO SAML profile? To me it seems like you might mix the two different guides. The screenshot seems to be from the "SSO profile for your organization" but the xml file from "Third-party SSO SAML profile".

**Comment by EricCartman9969** [comment]
I've tried both of these methods, but they both give me an error. The other error is that metadata cannot find the instance

**Reply by mffap** [reply]
Not sure how to help you best with the information provided. Which of the both flows are you trying to achieve? Can you maybe share a screen recording that shows the flow and the errors? Or can you describe for the flow your configuration and where the errors are shown?

**Reply by hifabienne** [reply]
Can you please make a recording of the flow with the error happening? At the moment we can't help you as we do not have enough information. 
And can you please let us know if you want to configure SSO profile for your organization or third-party sso saml profile.
And send us the according configuration.

We can only help you if we know what you want to do, and what you already have done.

**Reply by EricCartman9969** [reply]
i want to configure SSO profile for your organization

Set up SSO with third-party identity provider | Enable (check)
Sign-in page URL | mydomain/saml/v2/SSO
Sign-out page URL |mydomain/saml/v2/SLO
Verification Certificate | Upload the certificate (idp.crt)
Use a domain-specific issuer | Enable (check)
Network masks | Leave blank
Change password URL | mydomain/ui/console/users/me?id=security

Use this config google workspace return me ::This account cannot be accessed because the login credentials could not be verified

**Reply by mffap** [reply]
what is the contents of the xml file?
